doc/organization.md

```diff
@@ -1,14 +1,18 @@
 # Organization
 
-- Front-End Development:
+## Front-End Development:
+Front-End Development refers to the gui components of the project. The look and feel of the project is handled in this role. 
 
   -- Jonasty
 
-- Back-End Development: 
+## Back-End Development: 
+Back-End Development refers to the inner workings--the handling of various kinds of fetch requests and algorithmic solutions. 
 
   -- Chris Pruett
 
-- Database
+## Database
+The Database role is responsible for storing, altering, and removing employee records from the database.
+
   -- Dylan
   -- Joshua Carrier (*)
```

Comment:
What is the (*) for?

Reply:
It's a wildcard marker. Not sure if we agreed on you having a sort of wildcard role.